pages/e/[eventId]/index.tsx

```
@@ -656,34 +652,37 @@ export default function Event(props) {
+}) {
+  try {
+    const event = await fetchEventById(params.eventId);
 
-  if (!event || !event.id) {
+    return {
+      props: {
+        event,
+      },
+      revalidate: 60,
+    };
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)